Linphone/Core/ICoreHttpClient.cs

```diff
@@ -0,0 +1,27 @@
+﻿using BelledonneCommunications.Linphone.Presentation.Dto;
+using System;
+using System.Threading.Tasks;
+
+namespace BelledonneCommunications.Linphone.Core
+{
+    public interface ICoreHttpClient
+    {
+        Task<CallsCommandServiceInitiateIncomingResponse> InitiateIncomingCallAsync(string callerPhoneNumber, string agentPhoneNumber, string inboundService);
```

Comment:
Please consider separation of concern principle, and make related APIs to multiple interfaces

Reply:
I suggest you to create two interface, one for calls, and another for agents

Please make well naming interfaces and classes based on responsibilities

Reply:
It's used on our call center domain and customized to the customer needs.
we do not need this here and i will remove it from CallViewModel.

Reply:
Resolved.